### ISSUE TITLE
This app cannot be installed on this workspace. Apps with this feature are only available to Enterprise customers.

### ISSUE THREAD

**Issue by Flimm** [sgratzl/slack_cleaner2#69]
When trying to create and install the Slack app into a workspace, I get this error message:

> This app cannot be installed on this workspace. Apps with this feature are only available to Enterprise customers.

**Comment by sgratzl** [comment]
does it specify which feature, permission is the enterprise one?

**Comment by Flimm** [comment]
No, it doesn't unfortunately. I added the ones mentioned in the README.

**Comment by ebisbe** [comment]
I set up the wrong permissions and faced the same issue after updating them. I ended up recreating the app and then it all worked.
